src/function/algebra/resolve.js

```diff
@@ -35,11 +35,12 @@ export const createResolve = /* #__PURE__ */ factory(name, dependencies, ({
    *
    *     simplify, evaluate
    *
-   * @param {Node} node
-   *     The expression tree to be simplified
+   * @template nodeOrNodes
+   * @param {nodeOrNodes} node
+   *     The expression tree (or trees) to be simplified
    * @param {Object} scope
    *     Scope specifying variables to be resolved
-   * @return {Node} Returns `node` with variables recursively substituted.
+   * @return {numOrStr extends Node[] ? Node[] : Node} Returns `nodeOrNodes` with variables recursively substituted.
    * @throws {ReferenceError}
```

Comment:
I don't understand the significance of the identifier "numOrStr" here -- did you mean "nodeOrNodes"?

Reply:
Shoot, yes I did. Bad copy n paste 🙄